src/ast_function_definition.cpp

```diff
@@ -1,13 +1,38 @@
 #include "ast_function_definition.hpp"
 
-FunctionDefinition::FunctionDefinition(Node* declaration_specifiers, Node* declarator, Node* compound_statement) {
-  branches.insert(branches.end(), {declaration_specifiers, declarator, compound_statement});
+void FunctionDefinition::emitRISC(std::ostream &stream, Context &context) const
+{
+    // TODO: this is not complete.
+    std::cerr << "FunctionDefinition: emitRISC is not fully implemented." << std::endl;
+
+    // Emit assembler directives.
+    // TODO: these are just examples ones, make sure you understand
+    // the concept of directives and correct them.
+    stream << ".text" << std::endl;
+    stream << ".globl f" << std::endl;
+
+    this->declarator->emitRISC(stream, context);
+
+    if (this->compoundStatement != nullptr)
+    {
+        this->compoundStatement->emitRISC(stream, context);
+    }
 }
 
-void FunctionDefinition::emitRISC(std::ostream &stream, Context &context) const {
-  // Emit declarator
-  branches[1]->emitRISC(stream, context);
+void FunctionDefinition::print(std::ostream &stream) const
+{
+    // TODO: this is not complete.
+    std::cerr << "FunctionDefinition: print is not fully implemented." << std::endl;
```

Comment:
If we're putting this reminder, shouldn't it appear in every single `print` function we provide?